pipeline_dsl/test/test_job.py

```diff
@@ -73,10 +74,11 @@ def test_get_put(self):
             job.get("res-1", False, ["testjob"], params={"test": 1})
             job.get("res-2", True, ["testjob2"], params={"test": 2})
 
-            job.put("res-1", {"test": 3})
+            job.put("res-1", {"test": 3}, get_params={"test": True})
             job.put("res-2", {"test": 4})
 
             obj = job.concourse()
+            self.maxDiff = None
```

Comment:
This is related to the `assertDictEqual`, isn't it?  It looks a bit out of place there to me. Either at the start of the test, for each test or directly before the `assert`. It looks like it is related to `job.concourse()`
WDYT?

Reply:
Ups, this was just a leftover from the debugging. I think we should either use it everywhere or nowhere so I would just remove it from this PR.

Reply:
We removed it in https://github.com/SAP/pipeline-dsl/commit/1c6a64cd9d17ab6bb98921f4dfaa1815b7da100f